### DISCUSSION TITLE
Visual Studio 2022, ASP.NET Core 6 with PeachPie-Wordpress NuGet package - blank page in browser

### DISCUSSION THREAD

**Discussion by biyanpian** [peachpiecompiler/peachpie#1049] [Q&A] [answered]
I am attempting to use the PeachPie-Wordpress NuGet package in Visual Studio 2022 (ASP.NET Core 6), with MariaDB and using Letsencrypt for the SSL certificate.  In general, the solution seems to be functioning.  When I start the project in Visual Studio, it runs correctly and loads the web content as expected.

I am using IIS to serve, and when I attempt to access the published site, all I see in the browser is a blank page (and blank page source).  The SSL certificate is recognized by the browser, and no errors are indicated.  The IIS log shows status 307 for port 80, indicating correctly the redirect to port 443.  The log subsequently shows status 200 for port 443, suggesting normalcy.

It has been a long time since I've set up hosting, and there are a number of new variables with this solution, so I expect and hope that I am missing something simple which would cause published content to load correctly in a browser.

Any suggestions?...or direction to another location where I might ask for help?

**Comment by jakubmisek** [comment, marked as answer]
Hi!

Please ensure you have updated to the recent release version `5.9.3-rc-002`. It should provide more details in case of an error.

If it won't help, debugging in Visual Studio (with disabled Tools -> Debug -> Enable Just My Code) should reveal an exception that's causing the issue.

I hope it helps!
